aerleon/lib/cisco.py

```diff
@@ -685,7 +690,7 @@ def _GetIpString(self, addr):
             return 'host %s' % (addr.network_address)
         return addr
 
-    def _FormatPort(self, port, proto):
+    def _FormatPort(self, port: Union[Tuple[()], Tuple[int, int]], proto: Union[int, str]) -> str:
```

Comment:
What is `Tuple[()]`? Is this a real type we want as input? Shouldn't it be None? Also, the docstring says that port can be "str list" but the type expression is `Tuple[int, int]` - they cannot both be right.

Reply:
It should be `Optional[Tuple[int, int]]` just looking at the code here and the code in `PortMap.GetProtocol`.

Reply:
The docstring is definitely incorrect.

Reply:
Yea the docstring is terrible looking at it. I can fix that up in this PR.
`Tuple[()]` is meant to be an empty tuple. 
Some select output from printing `port` and `type(port)`
```
E   + <class 'tuple'>
E   + ()
E   + <class 'tuple'>
E   + (1024, 65535)
E   + <class 'tuple'>
E   + ()
E   + <class 'tuple'>
E   + (1024, 65535)
E   + <class 'tuple'>
E   + ()
E   + <class 'tuple'>
E   + (1024, 65535)
E   + <class 'tuple'>
E   + ()
E   + <class 'tuple'>
```

I don't think it would be valid to do optional since `Optional` would be equal to a union of the type and None.
```
>>> tuple() == None
False
```
And those are not equal.

Reply:
Ok, that makes sense. I'll resolve the other suggestions to use Optional. I would suggest swapping the order from `Tuple[()], Tuple[int, int]` to `Tuple[int, int], Tuple[()]` for clarity.

Reply:
It would be nice if there was a type hint for "falsy" things!